python/simpler/worker.py

```
@@ -1690,8 +1730,269 @@ def handle_control(sub_cmd: int) -> tuple[int, str]:  # noqa: PLR0912 -- one bra
+                        prepared_frames[index] = identity
+                        _mailbox_store_i32(frame_state_addr, _BACKEND_READY)
+                        continue
+
+                    if frame_state == _ACTIVATE:
+                        prepared_identity = prepared_frames.get(index)
+                        if prepared_identity != identity:
+                            if prepared_identity is not None:
+                                try:
+                                    abort_prepared(prepared_identity)
+                                except Exception:  # noqa: BLE001
+                                    pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)